### ISSUE TITLE
VN type character model?

### ISSUE THREAD

**Issue by Rezar0Fura** [DALFC/dalsp#3]
Is it possible to have the character models of the inbuilt game as if they are like VN(Visual novel)? Like here. Along with the Backgrounds of the VN?

![Itsuka](https://github.com/DALFC/dalsp/assets/140922879/43d99667-7108-4c0b-8b2d-90d0af0eeb37)
![Date-a-Liv](https://github.com/DALFC/dalsp/assets/140922879/9d5e1121-63ec-48ff-918e-f5e49b7c40e6)
![maxresdefault](https://github.com/DALFC/dalsp/assets/140922879/5b2f8828-170a-465e-95e6-2a101f0632d7)

Basically, all the parts which make up the VN elements( The character sprites, the backgrounds, Game CG, the music, the voicelines). (Thought I am only interested in the character models of VN, if not with the BGs for now.)

![fa707106d4272cf3e78f6d354bfdf5957a6997dc2b07024f297b0d14eea9cf64_600](https://github.com/DALFC/dalsp/assets/140922879/aa8a4bf0-83b4-46d2-902e-ba42366909fa)

Its a heavy request, but hope you could help me.

**Comment by DALFC** [comment]
I didn't save Live2D models when I made this repo. Now the game has changed its game engine to Unity and so the file structures have changed too. I haven't played the game for a while but I believe you can find the .bundle files from the game's data and open it in AssetStudio to extract the resources you need.

**Comment by Rezar0Fura** [comment]
Alright, so after these months I got myself familiar with some coding and live 2d. Since the game got shut down in past, I want to find a new way to fetch the ".bundle" file or even better any other source for the 2d live models?

**Comment by DALFC** [comment]
The game Japan server is still going, you can get new assets there. https://github.com/respectZ/date-a-live/tree/master/live2d/assets also have some

**Comment by Rezar0Fura** [comment]
Thanks. What about the global version, if there is any?

**Comment by DALFC** [comment]
I don't know
